tests/by-util/test_split.rs

```diff
@@ -758,3 +758,36 @@ fn test_round_robin() {
     assert_eq!(file_read("xaa"), "1\n3\n5\n");
     assert_eq!(file_read("xab"), "2\n4\n");
 }
+
+#[test]
+fn test_split_invalid_input() {
+    // Test if stdout/stderr for '--lines' option is correct
```

Comment:
I don't understand what this comment is trying to say.

Reply:
bad copy and paste :)